### PR TITLE
Change default compressor to LZ4

### PR DESCRIPTION
**Pull request by Cpaulyz**
## Description

Change default compressor to LZ4 accoording to the experiment result in https://apache-iotdb.feishu.cn/docx/ThmHdPKw9oJv95xvLhWco2WnnLh

image

image

(The two images above are from Jinzhao Xiao, Yuxiang Huang, Changyu Hu, Shaoxu Song, Xiangdong Huang, Jianmin Wang. Time Series Data Encoding for Efficient Storage: A Comparative Analysis in Apache IoTDB. International Conference on Very Large Data Bases, VLDB, 2022. [[paper](https://sxsong.github.io/doc/22vldb-encoding.pdf)] )

### CODE REVIEW

File: integration-test/src/test/java/org/apache/iotdb/db/it/schema/IoTDBCreateAlignedTimeseriesIT.java [hunk 1]
```diff
@@ -94,7 +93,7 @@ public void testCreateAlignedTimeseries() throws Exception {
   public void testCreateAlignedTimeseriesWithDeletion() throws Exception {
     String[] timeSeriesArray =
         new String[] {
-          "root.sg1.d1.vector1.s1,DOUBLE,PLAIN,SNAPPY", "root.sg1.d1.vector1.s2,INT64,RLE,SNAPPY"
+          "root.sg1.d1.vector1.s1,DOUBLE,PLAIN,SNAPPY", "root.sg1.d1.vector1.s2,INT64,RLE,LZ4"
```
Comment (JackieTien97) [inline integration-test/src/test/java/org/apache/iotdb/db/it/schema/IoTDBCreateAlignedTimeseriesIT.java:96]: Why `root.sg1.d1.vector1.s1` still be  `SNAPPY`?

File: integration-test/src/test/java/org/apache/iotdb/util/AbstractSchemaIT.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public abstract class AbstractSchemaIT {
 
   @Parameterized.Parameters(name = "SchemaEngineMode={0}")
   public static Iterable<SchemaTestMode> data() {
-    return Arrays.asList(SchemaTestMode.Memory, SchemaTestMode.PBTree);
+    return Arrays.asList(SchemaTestMode.Memory);
```
Comment (JackieTien97) [inline integration-test/src/test/java/org/apache/iotdb/util/AbstractSchemaIT.java:60]: Why removing `PBTree` mode.

**Review by JackieTien97 — CHANGES_REQUESTED** [review]

File: integration-test/src/test/java/org/apache/iotdb/util/AbstractSchemaIT.java (hunk 2 above)
Comment (Cpaulyz) [reply integration-test/src/test/java/org/apache/iotdb/util/AbstractSchemaIT.java:60]: For local testing convenience. Already added back.

File: integration-test/src/test/java/org/apache/iotdb/db/it/schema/IoTDBCreateAlignedTimeseriesIT.java (hunk 1 above)
Comment (Cpaulyz) [reply integration-test/src/test/java/org/apache/iotdb/db/it/schema/IoTDBCreateAlignedTimeseriesIT.java:96]: Compressor=SNAPPY is used when creating root.sg1.d1.vector1.s1

**Review by JackieTien97 — APPROVED** [review]

**Review by JackieTien97 — APPROVED** [review]

**Pull request merged**